### ISSUE TITLE
Even with Rails 3.1 Installed, When installing engineX it will install the latest version of Rails 3.0

### ISSUE THREAD

**Issue by ardavis** [josevalim/enginex#35]
I installed Rails 3.1.0 into a new gemset, then proceeded to install enginex.

EngineX is not detecting that Rails 3.1 exists, and is installing Rails 3.0.10 in parallel. 

Just a heads up :)

Thanks,
Andrew Davis
NASA - KSC

**Comment by josevalim** [comment]
Something similar to enginex was merged into rails 3.1. Enginex per se does not work with rails 3.1.

**Comment by ardavis** [comment]
This is interesting. I wasn't made aware of that. Thanks.

**Comment by Confusion** [comment]
I think it would be helpful if you mentioned in the readme that this project is neither tested with, nor required for, Rails 3.1+. E.g. http://stackoverflow.com/questions/9053290/reusable-part-of-application points here and people using more recent Rails versions are better off reading the documentation at http://api.rubyonrails.org/classes/Rails/Engine.html
